### PR TITLE
Carnivore pemmican

### PR DESCRIPTION
**Pull request by anoobindisguise**
#### Summary
Content "Pemmican variant for carnivores"

#### Purpose of change
Pemmican is a useful survival food but it doesn't necessarily need to contain fruit or veggie matter. Some pemmican IRL is/was made of only fat and meat.

#### Describe the solution
Introduce ~~"pemmican't"~~ meat-only pemmican, a carnivore variant of pemmican. 

#### Describe alternatives you've considered
Make the name less stupid - name changed to Meat-Only Pemmican from "Pemmican't".

### CODE REVIEW

**Comment by descan** [comment]
Pemmicarn? Pemmeatcan?

**Comment by TheShadowFerret** [comment]
~~It's clearly Pemmicarni~~

**Comment by I-am-Erk** [comment]
I think pemmicant is a confusing name, it doesn't make me think "no fruit" it makes me think "doesn't last as long or something". I'd suggest just "meat pemmican", despite being boring AF

**Comment by PatrikLundell** [comment]
I don't think "meat pemmican" is good, as it implies ordinary pemmican is some vegan meatless fake. I'd rather go with something like "pemmican (meat only)", or "pemmican for carnivores". I don't think there's a need to keep the name very short, is there? Even if the name is cut off, the carnivore player will know that the pemmican in their inventory is the thing they need.

**Review by PatrikLundell — COMMENTED** [review]
Shouldn't it be "Contains no fruit or veggies" rather than "Has"?

**Comment by chrispikula** [comment]
What about pemmican (plant free) ?

We see x (meat free) as advertisement IRL today, so it would mesh that it's opposite would work well.

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -723,6 +723,31 @@
     "vitamins": [ [ "vitC", 3 ], [ "calcium", 5 ], [ "iron", 21 ] ],
     "fun": 3
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "pemmicant",
```
Comment (Fris0uman) [inline data/json/items/comestibles/meat_dishes.json:728]:
```suggestion
    "id": "pemmican_meat",
```
Having clear ids is also nice for future contributors

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -3070,6 +3070,33 @@
       ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "pemmicant",
```
Comment (Fris0uman) [inline data/json/recipes/recipe_food.json:3076]:
```suggestion
    "result": "pemmican_meat",
```

**Pull request merged**